src/assets/wise5/authoringTool/project-authoring-lesson/project-authoring-lesson.component.html

```diff
@@ -0,0 +1,19 @@
+<div id="{{ item.key }}" fxLayoutAlign="start center" class="pointer groupHeader">
+  <mat-checkbox
+    color="primary"
+    [(ngModel)]="item.checked"
+    (click)="selectNode()"
+    [disabled]="disableSelection"
```

Comment:
Emit event directly?
```suggestion
    (click)="selectNodeEvent.emit()"
```